geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-The Redis API for <%=vars.product_name%> allows <%=vars.product_name%> to function as a drop-in replacement for a
+The <%=vars.product_name%> API for Compatibility with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
 highly-available Redis data store, letting Redis applications take advantage of
-<%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
+<%=vars.product_name%>’s scaling capabilities without changing their client code. Compatible with Redis Clients connect to a <%=vars.product_name%>
 server in the same way they connect to a Redis server, using a hostname and a port number, with
 optional password authentication.
 
 <img src="../images/redis_api_for_geode.png" class="image" />
 
-## <a id="using-the-api" class="no-quick-link"></a>Using the Redis API for <%=vars.product_name%>
+## <a id="using-the-api" class="no-quick-link"></a>Using the <%=vars.product_name%> API Compatible with Redis
 
-The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis
-commands.
+The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Compatible with Redis commands.
 
 Use gfsh to start at least one server with a command of the form:
 
 ```pre
 start server \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --redis-port=<redisPort> \
-  --redis-bind-address=<redisBindAddress> \
-  --redis-password=<redisPassword>
+  --compatible-with-redis-port=<compatibleWithRedisPort> \
+  --compatible-with-redis-bind-address=<compatibleWithRedisBindAddress> \
+  --compatible-with-redis-password=<compatibleWithRedisPassword>
 ```
 
-If any of the options `redis-bind-address`, `redis-password`, or `redis-port` are included, a compatible-with-Redis Geode server will be started.
+If any of the options `compatible-with-redis-bind-address`, `compatible-with-redis-password`, or `compatible-with-redis-port` are included, a Geode server compatible with Redis will be started.
 
 Replace `<serverName>` with the name of your server.
 
 Replace `<locatorPort>` with your locator port.
 
-Replace `<redisPort>` with the port that the <%=vars.product_name%> server listens on for Redis commands. The typical port used with a Redis cluster is 6379.
+Replace `<compatibleWithRedisPort>` with the port that the <%=vars.product_name%> server listens on for Redis-compatible commands. The typical port used with a Redis-compatible cluster is 6379.
```

Comment:
Same as above, this could just be `Redis commands`.  Also, does the `compatible with` part need to come before Redis?  so `compatible-with-Redis cluster` (or something) vs. `Redis-compatible` cluster?